imports/plugins/core/checkout/client/container/cartDrawerContainer.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component, PropTypes } from "react";
+import { Session } from "meteor/session";
+import { Cart, Media } from "/lib/collections";
+import { Reaction } from "/client/api";
+import { composeWithTracker } from "/lib/api/compose";
+import { Loading } from "/imports/plugins/core/ui/client/components";
+import CartDrawer from "../components/cartDrawer";
+
+class CartDrawerContainer extends Component {
+  static propTypes = {
+    defaultImage: PropTypes.object,
+    lowInventory: PropTypes.bool,
+    productItems: PropTypes.array
+  }
+
+  handleImage(item) {
+    const { defaultImage } = item;
+    if (defaultImage && defaultImage.url({ store: "small" })) {
+      return defaultImage;
+    }
+    return false;
+  }
+  /**
+  * showLowInventoryWarning
+  * @param {Object} productItem - product item object
+  * @return {Boolean} return true if low inventory on variant
+  */
+  showItemLowInventoryWarning(productItem) {
+    const { variants } = productItem;
+    if (variants && variants.inventoryPolicy &&
+      variants.lowInventoryWarningThreshold) {
+      return variants.inventoryQuantity <=
+        variants.lowInventoryWarningThreshold;
+    }
+    return false;
+  }
+
+  handleLowInventory = (productItem) => {
+    return this.showItemLowInventoryWarning(productItem);
+  }
+
+  handleShowProduct = (productItem) => {
+    if (productItem) {
+      Reaction.Router.go("product", {
+        handle: productItem.productId,
+        variantId: productItem.variants._id
+      });
+    }
+  }
+
+  pdpPath(productItem) {
+    if (productItem) {
+      const handle = productItem.productId;
+      return Reaction.Router.pathFor("product", {
+        hash: {
+          handle,
+          variantId: productItem.variants._id
+        }
+      });
+    }
+  }
+
+  handleRemoveItem(event) {
+    event.stopPropagation();
+    event.preventDefault();
+    const currentCartItemId = event.target.getAttribute("id");
+    $(`#${currentCartItemId}`).fadeOut(500, () => {
```

Comment:
Jquery is exported globally by Meteor, but it's probably a good idea to import as that might not always be the case, but I feel like we'd rather just not use jquery,  I thought we were deprecating this syntax in favor of CSS only...